packages/metal-dom/src/domNamed.js

```diff
@@ -205,21 +205,6 @@ export function buildFragment(htmlString) {
 	return fragment;
 }
 
-/**
- * Checks if the first element contains the second one.
- * @param {!Element} element1
- * @param {!Element} element2
- * @return {boolean}
- */
-export function contains(element1, element2) {
-	if (isDocument(element1)) {
```

Comment:
Hey @jbalsas, It would be valid to start issuing `warn` messages talking about deprecated. When the developer is using this feature. I think will be much more aware when you see this on the console when are developing your applications.

Something like proposed in #364.